cmd/operator/deploy/crds/monitoring.googleapis.com_clusterpodmonitorings.yaml

```diff
@@ -293,6 +293,8 @@ spec:
     subresources:
       status: {}
   - name: v1alpha1
+    deprecated: true
+    deprecationWarning: Use version v1
```

Comment:
I'd recommend omitting `deprecationWarning` and using the standard warning message that recommends v1

Reply:
Great suggestion! Done.